examples/integrations/vault/hcp-reactive/src/main/resources/application.yaml

```diff
@@ -50,6 +50,10 @@ vault:
       k8s:
         enabled: false
       app-role:
+        # this is not needed, as we use a builder,
```

Comment:
Not sure why this change (and related changes in code) is shown as part of your PR.